db/sqlinterfaces.c

```diff
@@ -2823,7 +2823,7 @@ static int post_sqlite_processing(struct sqlthdstate *thd,
                 write_response(clnt, RESPONSE_ROW_LAST_DUMMY, 0, 0);
             }
         } else {
-            if (postponed_write) {
+            if (postponed_write && !clnt->send_one_row) {
```

Comment:
Can you please add a comment here to explain for future readers? I forget already why this is required.

Reply:
Will do.

Reply:
We're not skipping response and have a row to write and now this makes us skip the write?